src/components/Section3.vue

```diff
@@ -1,14 +1,121 @@
 <template>
-<div></div>
+  <div class="section3">
+    <div class="section3__text">
+      <h2>
+        Buy a set with a <strong class="yellow-text">30%</strong> discount
+      </h2>
+      <p>
+        These baby filters make your drone look faster. Or at least you won’t
+        lose them like those other black filters
+      </p>
+    </div>
+    <div class="section3__slider">
+      <div class="section3__slider-slide">
+        <img
+          class="section3__slider-image"
+          alt="sorry"
+          src="../assets/slide1.png"
+        />
+        <h3>Fire engine red filters set</h3>
+        <p>
+          These baby filters make your drone look faster. Or at least you won’t
+          lose them like those other black filters
+        </p>
+        <p class="price">
+          240 USD
+          <span class="decorated">260 USD</span>
+        </p>
+        <button class="primary-button slide-button">Buy</button>
+      </div>
+      <div class="section3__slider-slide">
+        <img
+          class="section3__slider-image"
+          alt="sorry"
+          src="../assets/slide2.png"
```

Comment:
This images is a static resource (or dynamic data) so please replace it with
``` src="/products/slide2.png" ```

and generate folder with images `slide[...].png` on path ``` public/products ```